modules/custom/az_publication/config/schema/az_citation_style.schema.yml

```
@@ -11,5 +11,8 @@ az_publication.az_citation_style.*:
     style:
       type: string
       label: 'Style'
+    custom:
+      type: integer
```

I like @trackleft's suggestion to make this an actual boolean property instead of integer.
```suggestion
      type: boolean
```

---

Thanks @joeparsons ! I was on the fence about which way to go on this. For some reason `Drupal\Core\Render\Element\Checkbox` uses integer `0` an `1` as its normal values and not boolean (not sure why).

I do think an actual boolean in the config makes more sense - I can update this.